webserver/src/graph/redis.js

```
@@ -5,7 +5,9 @@ const socketio = require("../socketio");
 const redis = {}
 const listeners = {};
 
-const pipelines = ['concept-map', 'modify-graph', 'expand-material', 'concept-recommendation', 'resource-recommendation'];
+const pipelines = ['concept-map', 'modify-graph', 'expand-material', 'concept-recommendation', 'resource-recommendation',
+  , 'get_resources_by_main_concepts'
+];
```

Biome (1.9.4)

[error] This array contains an empty slot.

Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)